BMM.Tests/BMM.Core.Test/Unit/ViewModels/TrackInfoViewModelTests.cs

```diff
@@ -39,7 +39,7 @@ protected override void AdditionalSetup()
         public async Task SelectSection_ShouldNotCallOpenUriWhenThereIsNoExternalRelation()
         {
             // Arrange
-            var trackInfoViewModel = new TrackInfoViewModel(_uriOpenerMock.Object, ExceptionHandler.Object, _deepLinkHandlerMock.Object, TextResource.Object);
```

Comment:
do the unit tests still work? The bmmLanguageTextSource from the BaseViewModel is now null which could lead to problems in some cases, right?